ui/src/app/base/selectors/auth/auth.test.js

```diff
@@ -1,72 +1,69 @@
 import auth from "./auth";
+import { userState } from "testing/factories";
 
 describe("auth", () => {
   it("can get the current user details", () => {
     const state = {
-      user: {
+      user: userState({
```

Comment:
I'm not totally sure if these should all get updated as this state object will need to get replaced with the root state factory, but we can't do that until we have factories for all the state objects.